cg/services/post_processing/abstract_classes.py

```diff
@@ -38,8 +36,9 @@ def get_files_to_store(self, run_data: RunData) -> list[Path]:
 
 class PostProcessingMetricsParser(ABC):
 
+    @staticmethod
     @abstractmethod
-    def parse_metrics(self, metrics_paths: list[Path]) -> RunMetrics:
+    def parse_metrics(metrics_files: list[Path]) -> RunMetrics:
```

Comment:
Can we assume that all metrics will share at least one meaningful common parent directory? If so, maybe it is better to encapsulate the logic for identifying the individual report files in the concrete implementation of the parser

Reply:
It makes the interface for the metrics parser class leaky. Code using the parser will need to have logic for finding the paths and I think it would be better to encapsulate that in the metrics parser class.

Reply:
There is a RunFilemanager that handles the path constructions. It is not part of this PR.

Reply:
Ah, yeah maybe the metrics parser should just accept that in the constructor and handle it internally still

Reply:
implemented